### ISSUE TITLE
Zoom Integration Results in Accidental Zoom Calls

### ISSUE THREAD

**Issue by cbutler-affirm** [mattermost/mattermost-server#19281]
Per Mattermost guidelines, GitHub issues are for bug reports: <http://www.mattermost.org/filing-issues/>.

For troubleshooting see: http://forum.mattermost.org/.
For feature proposals see: http://www.mattermost.org/feature-requests/

If you've found a bug--something appears unintentional--please follow these steps:

1. Confirm you’re filing a new issue. [Search existing tickets in Jira](https://mattermost.atlassian.net/jira/software/c/projects/MM/issues/) to ensure that the ticket does not already exist.
2. Confirm your issue does not involve security. Otherwise, please see our [Responsible Disclosure Policy](https://about.mattermost.com/report-security-issue/).
3. [File a new issue](https://github.com/mattermost/mattermost-server/issues/new) using the format below. Mattermost will confirm steps to reproduce and file in Jira, or ask for more details if there is trouble reproducing it. If there's already an existing bug in Jira, it will be linked back to the GitHub issue so you can track when it gets fixed. 

#### Summary
The one-click "Start Zoom Meeting" button is easily clicked by accident when closing a message thread. The button is directly underneath the close thread "x" button. This results in a number of accidental Zoom meeting creations. It would be helpful to either offset the buttons or provide a configuration option to confirm that you really intended to start a Zoom meeting.

#### Steps to reproduce
1- Enable Zoom integration
2- Open message thread
3- Click "x" to close thread
4- Click again without moving mouse

#### Expected behavior
No unexpected Zoom meetings. Change the alignment of the buttons or provide a configurable option that provides a confirmation before starting the meeting.

#### Observed behavior (that appears unintentional) 
<img width="542" alt="Screen Shot 2022-01-05 at 1 45 21 PM" src="https://user-images.githubusercontent.com/92941918/148286872-935d1bb0-2d9c-415c-828f-298c06c2fbcb.png">

#### Possible fixes
If you can, link to the line of code that might be responsible for the problem

**Comment by amyblais** [comment]
@cbutler-affirm What Mattermost server version are you on? We fixed a bug related to this in v6.2.

**Comment by cbutler-affirm** [comment]
Thanks @amyblais. I didn't see anything in the release notes that looked like an obvious fix for this. Which one are you referring to?

**Comment by amyblais** [comment]
@cbutler-affirm Here is a link to the ticket: https://mattermost.atlassian.net/browse/MM-39745
And link to pull request: https://github.com/mattermost/mattermost-webapp/pull/9424

**Comment by cbutler-affirm** [comment]
Thank you!
